tests/test_crs/valid_fiod.yaml

```diff
@@ -28,6 +30,7 @@ spec:
       read_ramp_time: 1
       filesize: 10MiB
       log_sample_rate: 1000
+      storagesize: 16Mi
```

Comment:
I think we don't need this variable

Reply:
the issue here is that we do not want fio to write to all of the PVC, by setting a larger storagesize(PVC size) and keeping filesize to 10MiB, we ensure that its not 100% full and future write operations will not fail.

Reply:
Thanks for the explanation @acalhounRH, that makes complete sense